urllib3/_collections.py

```diff
@@ -278,10 +268,7 @@ def __repr__(self):
     def _copy_from(self, other):
         for key in other:
             val = other.getlist(key)
-            if isinstance(val, list):
-                # Don't need to convert tuples
-                val = list(val)
-            self._container[key.lower()] = [key] + val
+            self._container[key.lower()] = [key] + list(val)
```

Comment:
@yetingsky a `HTTPHeaderDict` is not the only thing we might be copying from though. I don't trust other implementations to be 100% correct.